sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_util.py

```diff
@@ -221,5 +228,34 @@ def unnest_to_json(cand):
 
 
 def millis_to_timestamp(millis):
+  # type: (int) -> Timestamp
   micro_second = millis * 1000
   return Timestamp(micros=micro_second)
+
+
+def get_counter_metric(result, namespace, name):
+  # type: (PipelineResult, str, str) -> int
+  metrics = result.metrics().query(
+      MetricsFilter().with_namespace(namespace).with_name(name))
+  counters = metrics['counters']
+  if len(counters) > 1:
+    raise ValueError(
```

Comment:
ValueError is normally used to say the function argument being passed is invalid, I think we can use RuntimeError instead.

Reply:
thanks!!